docs/components/board/upboard.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Configure an upboard board"
+linkTitle: "upboard"
+weight: 70
+type: "docs"
+description: "Configure an upboard board."
+images: ["/icons/components/board.svg"]
+tags: ["board", "components"]
+# SMEs: Susmita
+---
+
+Configure an `upboard` board to integrate an Intel-based board like the [UP4000](https://github.com/up-board/up-community/wiki/Pinout_UP4000) into your robot.
+
+## Set up requirements
+
+Flash your Intel-based board with:
+
+1. The Ubuntu `"Server install image"` version of the operating system. For example, `ubuntu-22.04.2-live-server-amd64.iso`.
+Follow [these instructions](https://ubuntu.com/core/docs/install-nuc) to do so.
```

Comment:
Let's remove the link, the steps are kind of unclear. instead use this link https://ubuntu.com/tutorials/install-ubuntu-server#1-overview 
sorry for not sending you this earlier.

Reply:
Oh ok amazing no worries! I just couldn't find a better one originally